src/elm/Page/Shop.elm

```diff
@@ -482,6 +486,71 @@ view _ _ shared model _ =
             ]
 
 
+nameFromUserType : List UserProfile -> UserType -> Maybe String
+nameFromUserType profiles userType =
+    profiles
+        |> List.Extra.find (.userType >> (==) userType)
+        |> Maybe.map (.name >> langString)
+
+
+nameFromFareProduct : List FareProduct -> String -> Maybe String
+nameFromFareProduct products productId =
+    products
+        |> List.Extra.find (.id >> (==) productId)
+        |> Maybe.map (.name >> langString)
+
+
+stringFromStart : Model -> Maybe String
+stringFromStart model =
+    if model.now then
+        Just "Nå"
+
+    else
+        Maybe.andThen (Util.Format.isoStringToFullHumanized model.zone) model.isoTime
+
+
+stringFromZone : List TariffZone -> Model -> Maybe String
+stringFromZone tariffZones model =
+    let
+        findName zone =
+            tariffZones
+                |> List.Extra.find (.id >> (==) zone)
+                |> Maybe.map (.name >> langString)
+                |> Maybe.withDefault "-"
+
+        fromZoneName =
+            findName model.fromZone
+
+        toZoneName =
+            findName model.toZone
+    in
+        if model.fromZone == model.toZone then
+            Just <| "Reise i 1 sone (" ++ fromZoneName ++ ")"
```

Comment:
Should this be "Reise i én sone (..)"?

Reply:
Dette er samme som app foreløpig. Usikker på hva som er mest tydelig for ulike folkegrupper her. Kanskje @magnusolderoesaether har noen tanker?

Reply:
Stemmer for at vi har det samme som app foreløpig og heller revurderer som en del av copy